### PR TITLE
MongoDB ODM 1.0.5 release

### PR DESCRIPTION
**Pull request by malarzm**

### CODE REVIEW

File: site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      "require": {
+          "doctrine/mongodb-odm": "^1.0.5"
+      }
+  }
+
+Doctrine MongoDB ODM and PHP 7
+------------------------------
+
+While ODM still relies on old Mongo driver and no dates are scheduled for the 2.0 release,
+`it is already possible to run ODM with PHP 7! <https://twitter.com/alcaeus/status/697659616172359680>`__
```
Comment (alcaeus) [inline site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst]: I would add a note about this being possible with the upcoming 1.1 release of ODM and 1.3 of doctrine/mongodb, not the 1.0 and 1.2 releases, respectively. This should avoid confusion ;)

**Comment by alcaeus** [comment]
:+1:

File: site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst (hunk 1 above)
Comment (malarzm) [inline site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst]: Added in next line :)

File: site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  {
+      "require": {
+          "doctrine/mongodb-odm": "^1.0.5"
+      }
+  }
+
+Doctrine MongoDB ODM and PHP 7
+------------------------------
+
+While ODM still relies on old Mongo driver and no dates are scheduled for the 2.0 release,
```
Comment (jmikola) [inline site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst]: I would rephrase as "legacy MongoDB driver (i.e. `ext-mongo`)" and link the extension name to https://pecl.php.net/package/mongo

File: site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst (hunk 1 above)
Comment (jmikola) [inline site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst]: Suggestion: "it is possible to run ODM's development branch with the new MongoDB driver (i.e. `ext-mongodb`) on PHP 7 and HHVM! (see: this tweet)". I'd link `ext-mongodb` similar to the first part of the sentence, which is why I moved the tweet link to parens.

File: site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+`it is already possible to run ODM with PHP 7! <https://twitter.com/alcaeus/status/697659616172359680>`__
+(please note that for now you need to be running ``dev-master`` version of both ODM and the underlying
+``doctrine/mongodb`` library, this will change as soon as we release 1.1 and 1.3 versions respectively).
```
Comment (jmikola) [inline site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst]: "The new driver should be properly supported once we release versions 1.1 and 1.3 of the ODM and underlying Doctrine MongoDB library, respectively." I suggest linking "Doctrine MongoDB" to the GitHub repo.

File: site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+This is all possible thanks to our Andreas Braun (`@alcaeus <https://twitter.com/alcaeus>`__) work on
```
Comment (jmikola) [inline site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst]: "thanks to Andreas Braun's"

File: site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+`mongo-php-adapter <https://github.com/alcaeus/mongo-php-adapter>`__ which provides ext-mongo interface
+using new `mongo-php-library <https://github.com/mongodb/mongo-php-library>`__. If you can't wait to
+give ODM a test flight on PHP 7 now is the time! Also if you meet Andreas be sure to buy him a beer :)
```
Comment (jmikola) [inline site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst]: Suggestion (feel free to add links, of course): "`mongo-php-adapter`, which implements `ext-mongo`'s API atop `ext-mongodb` and `mongodb-php-library`."

File: site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+.. author:: Maciej Malarz <malarzm@gmail.com>
+.. categories:: none
+.. tags:: none
```
Comment (jmikola) [inline site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst]: Do we use tags for ODM posts?

Comment (malarzm) [inline site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst]: I'm not sure we use tags at all :)
